mogwai/src/view/dom.rs

```diff
@@ -849,12 +849,15 @@ impl<T: IsDomNode + AsRef<HtmlElement>> StyleView for View<T> {
 impl<T: IsDomNode> PostBuildView for View<T> {
     type DomNode = T;
 
-    fn post_build(&mut self, tx: Transmitter<T>) {
-        let t: Ref<T> = Ref::map(self.internals.borrow(), |internals| {
-            internals.element.unchecked_ref()
-        });
-        let t: T = t.clone();
-        tx.send_async(async move { t });
+    fn post_build(&mut self, _tx: Transmitter<T>) {
```

Comment:
Can you change this to `tx` since it is being used? I usually reserve `_*` for vars that are never used after being declared.

Reply:
Sure, I changed this because the compiler gives me a warning when running `cargo test` and via rust analyzer when editing. Both of which are because those aren't using the wasm32 architecture and so in that case `tx` isn't used.

Reply:
Oh, right - gotcha. Feel free to keep it in then!